hack/FAST

```diff
@@ -0,0 +1 @@
+1
```

Comment:
What does this file do?

Reply:
Just a way to bump the `wire-server-pr-fast` pipeline and trigger it since the `build` folder where the Dockerfile is is ignored.